101-azurefirewall-create-with-zones/azuredeploy.json

```diff
@@ -0,0 +1,185 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "virtualNetworkName": {
```

Comment:
add a defaultValue

Reply:
We don't have it the original template in 101-azurefirewall-create/azuredeploy.json so I kept it same way. Should I still add a default value here?

Reply:
yes, will make it easier for users to deploy

Reply:
Done.